airsonic-main/src/main/java/org/airsonic/player/service/search/IndexManager.java

```diff
@@ -187,7 +187,7 @@ private void stopIndexing(IndexType type) {
         boolean isUpdate = false;
         // close
         try {
-            isUpdate = -1 == writers.get(type).commit();
+            isUpdate = -1 != writers.get(type).commit();
```

Comment:
It's a simple code mistake.
The judgment as to whether or not a commit has been made is incorrect.
This fix allows Sercher refresh to work correctly.
Corrects the operation when multiple scans are executed, which is not in a single case.